packages/block-editor/src/store/selectors.js

```diff
@@ -2039,13 +2039,52 @@ export const getInserterItems = createRegistrySelector( ( select ) =>
 
```

Comment:
`getInserterItems` is a public selector, so we might have to create a private selector and make changes to that one instead 🤔

Reply:
Why? Is this change bound to break anything? It should improve the UX across the board. The normal classic functionality remains as is.

Reply:
If you use `getInserterItems` it will now show all blocks, but with a `rootClientId` property, so the handling is different, technically a breaking change.

Reply:
Made it a private option in [9a3f379](https://github.com/WordPress/gutenberg/pull/62169/commits/9a3f3791afca5000ee447bb1de6a52991a7eae71)